### PR TITLE
Add getNftContractBySymbol, getNft, update nft api queries.

### PR DESCRIPTION
**Pull request by liayoo**
Related changes: https://bitbucket.org/comcomai/nft-server/pull-requests/70/add-get-nft-by-symbol-api-update-nft-api
Related issue: https://www.notion.so/comcom/bot-nft-80829e73a5cf4e198558d6731cd9fb5d

### CODE REVIEW

File: src/asset.ts [hunk 1]
```diff
@@ -4,17 +4,72 @@ import { HttpMethod } from './types';
...
+        throw e.response.data;
+      });
+  }
+
   async getUserNftList(
     appId: string,
     chainId: string,
     ethAddress: string,
-    contractAddress?: string
+    contractAddress: string,
```
Comment (ehgmsdk20) [inline src/asset.ts:66]: It would be good to discuss whether to receive the contract address as an option.

Comment (liayoo) [reply src/asset.ts]: 🙆‍♀️ discussed on the other pr & updated to keep this optional

**Review by ehgmsdk20 — APPROVED** [review]
LGTM, thanks!
But since I am adding credit related functions to ainftJs, is it okay if I upgrade npm version after my pr merged?

**Comment by liayoo** [comment]
> 

Thanks for the review! And of course. We might want to check with @woomurf as well before a release, since he's working on the get & update activity APIs

**Pull request merged**